### PR TITLE
Final adjustments

### PR DESCRIPTION
**Pull request by chrsbell**
can I be done yet

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -223,47 +221,54 @@ class App extends React.Component {
...
+        }, Number(Animation.dimDuration))
+      });
+    });
   }
 
   render() {
-
     // pop-up review modal
-    const ReviewModal = this.state.showModal ? (<StyledAppModal ref={this.modal} reviews={this.state.reviews} ratings={this.state.ratings} close={this.closeModal.bind(this)} />) : null;
+    const ReviewModal = this.state.showModal
```
Comment (emmanuelmota) [inline client/src/components/App.jsx:231]: I'm curious as to what this line is doing. Im sure there is a reason for it.
Usually mine would like something like:
const  { showModal } = this.state;

File: client/src/components/AppModal.jsx [hunk 2]
```diff
@@ -124,14 +124,14 @@ class AppModal extends React.Component {
               </X>
             </CloseButton>
           </Container>
-          <StyledRatingOverview average={this.state.ratings.average} numReviews={this.state.reviews.length} isModal={true}/>
-          <StyledRatingGraphs ratings={this.state.ratings} isModal={true}/>
+          <StyledRatingOverview average={this.state.ratings.average} numReviews={this.state.reviews.length} isModal={true} />
```
Comment (emmanuelmota) [inline client/src/components/AppModal.jsx:127]: Style guide will ask you to always  de-structure these.

**Review by emmanuelmota — APPROVED** [review]

File: client/src/components/App.jsx (hunk 1 above)
Comment (chrsbell) [reply client/src/components/App.jsx:231]: good point, I will change it to what you suggested. The line is supposed to hold the modal component if the state says it's supposed to

File: client/src/components/AppModal.jsx (hunk 2 above)
Comment (chrsbell) [reply client/src/components/AppModal.jsx:127]: Will do!

**Pull request merged**